packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -43,13 +48,23 @@ export const _videoGalleryRemoteParticipantsMemo = (
         })
         .map((participant: RemoteParticipantState) => {
           const state = _isRingingPSTNParticipant(participant);
+          let displayName = participant.displayName;
+          /* @conditional-compile-remove(hide-attendee-name) */
+          if (isHideAttendeeNamesEnabled && participant.role === 'Attendee') {
```

Comment:
This logic is in both participantlistselector and videogalleryselector, to ensure these always are consistent, extract to a helper function, e.g., `maskDisplayNameWithRole(displayName, localUserRole, participantRole)`

Reply:
done